src/RoslynAnalyzers/Utilities/FlowAnalysis/FlowAnalysis/Analysis/ValueContentAnalysis/ValueContentAbstractValue.cs

```diff
@@ -171,7 +171,7 @@ internal ValueContentAbstractValue Merge(ValueContentAbstractValue otherState)
                 throw new ArgumentNullException(nameof(otherState));
             }
 
-            ImmutableHashSet<object?> mergedLiteralValues = LiteralValues.AddRange(otherState.LiteralValues);
+            ImmutableHashSet<object?> mergedLiteralValues = ImmutableHashSetExtensions.AddRange(LiteralValues, otherState.LiteralValues);
```

Comment:
Do you use any semantic search to find opportunities like this or are you just scouring code?